tomviz/python/tomviz/utils.py

```diff
@@ -47,24 +47,23 @@ def set_scalars(dataobject, newscalars):
     do.PointData.SetActiveScalars(name)
 
 
-def get_array(dataobject):
+def get_array(dataobject, order='F'):
     scalars_array = get_scalars(dataobject)
-    scalars_array3d = np.reshape(scalars_array, (dataobject.GetDimensions()),
-                                 order='F')
+    if order == 'F':
+        scalars_array3d = np.reshape(scalars_array,
+                                     (dataobject.GetDimensions()),
+                                     order=order)
+    else:
+        scalars_array3d = np.reshape(scalars_array,
+                                     (dataobject.GetDimensions()[::-1]),
+                                     order=order)
     return scalars_array3d
 
 
 def set_array(dataobject, newarray, minextent=None):
-    # Ensure we have Fortran ordered flat array to assign to image data. This
-    # is ideally done without additional copies, but if C order we must copy.
-    if np.isfortran(newarray):
-        arr = newarray.reshape(-1, order='F')
-    else:
-        print('Warning, array does not have Fortran order, making deep copy '
-              'and fixing...')
-        tmp = np.asfortranarray(newarray)
-        arr = tmp.reshape(-1, order='F')
-        print('...done.')
+    # Ensure we have flat array to assign to image data. Account for Fortran vs.
+    # C order.
+    arr = newarray.reshape(-1, order='A')
```

Comment:
So reading the documentation, the 'A' ordering doesn't change the underlying array, but instead adapts the indexing into the array? It will be a new view object where possible, so what does this end up doing at the VTK level?